### ISSUE TITLE
Announce movement when moving text with alt+shift+up or down arrows in Word

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5945]
In Word you can press alt+shift+up or down arrows to move a paragraph (or selected paragraphs) up or down in your document.  NVDA does not announce anything when using this command.

**Comment by michaelDCurran** [comment]
I assume it just reorders the paragraphs? If so... I wonder what we should actually announce. I'd think something like "moved above x" or "moved below x" where x is the content of the other paragrahph?

**Comment by nvaccessAuto** [comment]
Incubated in 97c161fdd55720fc531d43bf554ffacc1979b156.
